pkg/bootkube/status.go

```diff
@@ -25,7 +25,8 @@ const (
 	doesNotExist = "DoesNotExist"
 )
 
-func WaitUntilPodsRunning(pods []string, timeout time.Duration, selfHostedEtcd bool) error {
+// TODO: Don't pass the etcdServiceIP
```

Comment:
Sure. 
@pbx0 is CI exercising this currently? That's the one that needs to be manually triggered?